typespec-tests/tsp/internal.tsp

```diff
@@ -26,35 +26,46 @@ model ResponseInternal {
   property: ResponseInternalInner;
 }
 
+@access(Access.public, "python")
 model ResponseInternalInner {
   name: string;
 }
 
 // Response will be generated as output-only
-@include
+@access(Access.public)
 model Response {
   name: string;
 }
 
-// StandAloneData and StandAloneDataInner will be generated
-@include
+// StandAloneData will be generated as output
+@access(Access.public)
+@usage(Usage.output, "java")
```

Comment:
What is the difference between output-only model and default model?

Reply:
At present, no. output-only is currently the least possible model we can generate (it does not have a public ctor and setter).

But it may change, if there be difference of model of "none" and model of "output-only".